docs/CHANGELOG.md

```diff
@@ -58,6 +58,7 @@ The format is based on [Keep a Changelog][kc], and this project adheres to
 * The z in UiTransformBuilder now defaults to 1 instead of 0, allowing to skip defining the z in the ui prefabs. ([#946])
 * Added comments to ui prefab. ([#946])
 * Summarized all `use amethyst::` statements to allow collapsing in IDE's. ([#974])
+* Modified how the `Application` determines what events get sent to the `State`s ([#996])
```

Comment:
I would be more explicit here instead of saying that it changed prob better to say what changed.